doc/user/content/release-notes.md

```diff
@@ -102,6 +102,13 @@ Wrap your release notes at the 80 character mark.
 - Avoid panicking if a record in a regex-formatted source fails to decode
   as UTF-8 {{% gh 5008 %}}.
 
+- Validate `WITH` clauses in `CREATE SOURCE` and `CREATE SINK` statements. This
+  is a potentially backwards-incompatible change as previously any invalid
+  `WITH` clauses would be silently ignored, thus upgrading materialize in-place
```

Comment:
I would maybe just phrase this as `would be silently ignored. Upgrading Materialize in place ...` but feel free to tioli

Reply:
Ah! That's much better, thank you